src/trio/_tests/test_socket.py

```diff
@@ -648,11 +648,20 @@ async def res(
                 )
                 netlink_sock.close()
 
-            with pytest.raises(ValueError):
+            with pytest.raises(
+                ValueError,
+                match=r"^address should be a \(host, port(, \[flowinfo, \[scopeid\]\])*\) tuple$",
```

Comment:
could save this in a string variable and reuse it

Reply:
Changed in f89cd0a